api/app/models/env_canada.py

```diff
@@ -260,6 +329,7 @@ def process_model_run_urls(self, urls):
                                 # Flag the file as processed
                                 self.flag_file_as_processed(url)
                                 self.files_processed += 1
+                                logger.info('Processed %s', url)
```

Comment:
Hey I think this is redundant.
Go to this link:
https://console.pathfinder.gov.bc.ca:8443/console/project/auzhsi-dev/browse/pods/env-canada-hrdps-wps-api-pr-528-1602886020-6dhhm?tab=logs
and look at line 378

Reply:
Agree it's redundant. I've removed it now.